paddle/fluid/operators/math/complex_functors.h

```diff
@@ -66,7 +65,10 @@ using select_t = typename select<Head, Tail...>::type;
 template <typename T>
 using Real =
     select_t<cond<std::is_same<T, platform::complex64>::value, float>,
-             cond<std::is_same<T, platform::complex128>::value, double>, T>;
+             cond<std::is_same<T, platform::complex128>::value, double>,
```

Comment:
why complex64 and 128 are still needed here?

Reply:
There are still some other Ops need the Real and EnableComplex template definition. They will be remove after the others Ops finishing replacing the complex64/128 types.